### ISSUE TITLE
MS Sculpt Erognomic Desktop Mouse does not allow rebinding of right_opt

### ISSUE THREAD

**Issue by zxaos** [tekezo/Karabiner-Elements#668]
NB: This is the mouse from this bundle: https://www.microsoft.com/accessories/en-us/products/keyboards/sculpt-ergonomic-desktop/l5v-00001 and not the same as the Sculpt Comfort mouse from https://github.com/tekezo/Karabiner-Elements/issues/565

The mouse has a button that identifies in EventViewer as `right_option`:
![screen shot 2017-03-16 at 10 28 04](https://cloud.githubusercontent.com/assets/891202/24000991/480db5ce-0a33-11e7-91a0-03e28f45196d.png)

However it does not seem to be able to be modified by Karabiner. For example, the following simple modification appears to do nothing:

```JS
"simple_modifications": {
                "right_option": "vk_launchpad"
             },
```

**Comment by funkju** [comment]
+1

**Comment by ivan-m** [comment]
I think this is related to the changes made to fix #39
